### ISSUE TITLE
LC022 - False Positive, when return value is used.

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#254]
We have some code, where we want to get the value of the global language to reset the language of a report to the global language.

```al
trigger OnPostDataItem()
            var
            begin
               ...
                CurrReport.Language := GlobalLanguage; // This should lead to an error.
               ...
            end;
```
Everytime where we only use the return parameter of GlobalLanguage, there shouldn't be an info for Rule LC0022

**Comment by Arthurvdv** [comment]
Ho @pri-kise 

I think I found a way to improve the LC0022-rule based in your input. 
Could you help me and review if I have all scenario's covered in the examples below?

```
local procedure BuiltInGlobalLanguage()
var
    SomeInteger: Integer;
begin
    SomeInteger := GlobalLanguage;        // Should not raise warning LC0022 (allow to use for return value)
    SomeInteger := GlobalLanguage();      // Should not raise warning LC0022 (allow to use for return value)
    GlobalLanguage;                       // Should not raise warning LC0022 (allow to use for return value)
    GlobalLanguage();                     // Should not raise warning LC0022 (allow to use for return value)

    GlobalLanguage := 1337;               // Should raise warning LC0022
    SomeInteger := GlobalLanguage(1337);  // Should raise warning LC0022
    GlobalLanguage(SomeInteger);          // Should raise warning LC0022
end;
```

**Comment by pri-kise** [comment]
@Arthurvdv your suggestion looks good to me.
I always try to avoid the GlobalLanguage, but I collegeague of mine, did use the GlobalLanguage for a specific scenario to retrieve the current value, therefore your your suggestion looks good to me.

**Comment by Arthurvdv** [comment]
Hi, @pri-kise. With the https://github.com/StefanMaron/BusinessCentral.LinterCop/pull/256 change I believe this issue is resolved now and I'm closing this issue.
